src/test/java/com/app/store/BookControllerIntegrationTest.java

```diff
@@ -0,0 +1,73 @@
+package com.app.store;
+
+import com.app.store.entity.Book;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+@ExtendWith(SpringExtension.class)
```

Comment:
the ExtendWith is now part of the annotation SpringBootTest you can delete it from here